rotkehlchen/db/evmtx.py

```diff
@@ -216,21 +231,28 @@ def get_evm_transactions(
         """
         query, bindings = filter_.prepare()
         query, bindings = self._form_evm_transaction_dbquery(query, bindings, has_premium)
-        results = cursor.execute(query, bindings)
+        tx_map: dict[tuple[bytes, int], tuple[Any, ...]] = {}  # Group results by transaction (tx_hash + chain_id)  # noqa: E501
```

Comment:
I don't understand what you are doing here and why. You don't need to use chain id / tx_hash as key ... you have the unique transaction identifier.

Please simplify this